fw/http_parser.c

```diff
@@ -4553,8 +4574,23 @@ Req_Method_1CharStep: __attribute__((cold))
 	__FSM_METH_MOVE_finish(Req_MethUnloc, 'K', TFW_HTTP_METH_UNLOCK);
 
 	__FSM_STATE(Req_MethodUnknown, cold) {
-		__FSM_MATCH_MOVE_nofixup(token, Req_MethodUnknown);
```

Comment:
Now the macro isn't used and should be removed

Reply:
done